src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/prototype/SimpleStrandSwitchVariantDetector.java

```diff
@@ -67,7 +72,8 @@ public boolean test(final AlignedContig contig) {
 
     /**
      * Removes overlap from a designated alignment interval, so that the inverted duplicated reference span is minimal.
-     * If the two alignment intervals are NOT overlapping, return the original read.
+     * If the two alignment intervals are NOT overlapping, return the original aligned contig.
+     * For algorithm {@see <a href="https://github.com/broadinstitute/dsde-methods-sv/pull/8"}
```

Comment:
This is our private repo so I'm not sure we should reference it in comments in the public code -- it would just be frustrating for someone external.

Reply:
Agreed and removed. 
But we are more and more in need of a central place for documenting the algorithms used in the various stages of the pipeline.